setup.py

```diff
@@ -40,13 +32,12 @@
     "crcmod==1.7",
     "dill==0.2.6",
     "enum34==1.1.6",
-    "funcsigs==1.0.2",
     "future==0.16.0",
     "futures==3.1.1",
     "gapic-google-cloud-pubsub-v1==0.15.4",
     "google-apitools==0.5.11",
-    "google-auth==1.1.0",
-    "google-auth-httplib2==0.0.2",
+    "google-auth-httplib2==0.0.3",
```

Comment:
I would avoid changing any version in this PR, why did you change them? We  know that current version do work with the dataflow pipe.

Reply:
@enriquetuya I will left the versions with them original tags as you suggested. The changes of the tags were referenced in the doc attached but without airflow there is no need to update them.